e2e/nomostest/config_sync.go

```diff
@@ -78,9 +81,6 @@ var (
 
 	multiConfigMaps = filepath.Join(baseDir, "e2e", "raw-nomos", Manifests, multiConfigMapsName)
 
-	// clusterRoleName is the ClusterRole used by Namespace Reconciler.
-	clusterRoleName = fmt.Sprintf("%s:%s", configsync.GroupName, core.NsReconcilerPrefix)
```

Comment:
Will having multiple tests create & delete this role cause problems in a shared test env? I think that's why they had different name, right?

Reply:
The scaffolding already handles the lifecycle of this cluster role between tests (both for shared/ephemeral test environments). I'm basically just changing the name here so instead of creating the ns reconciler cluster role shipped with Config Sync we instead create one with a new name. Note this was essentially already a constant before (`configsync.gke.io:ns-reconciler`). I haven't tested it with a shared environment but I expect it should work

Reply:
Nevermind. SharedTestEnv runs in serial, so this isn't a problem.